nemo/collections/common/metrics/per.py

```diff
@@ -0,0 +1,343 @@
+import json
```

Comment:
@ssh-meister please add the following:
- header
- docstrings
- pytests
- comments on how to run/use the script (one the model is trained, during training etc)
 I haven't looked at the code yet

Reply:
sure!

Reply:
@ekmb, I've added the information that you have mentioned and pytests. Could I ask for a review, please?